Python/Product/TestAdapter/TestAnalyzer.cs

```diff
@@ -162,7 +162,7 @@ LocationInfo sourceLocation
             AnalysisValue classValue
         ) {
             return classValue.GetAllMembers(entry.Analysis.InterpreterContext)
-                .Where(v => v.Key.StartsWith("test"))
+                .Where(v => v.Key.StartsWith("test") || v.Key.StartsWith("runTest"))

```

Comment:
`StartsWith` is not correct for `runTest` - it must override the method, so it has to use that name exactly, unlike `test_...` pattern.